main.cc

```diff
@@ -482,7 +482,8 @@ struct gameb {
 		uint8_t padding[8];
 		int16_t player_index[24];
 
-		uint8_t misc000[76];
+		uint8_t misc000[75];
+		uint8_t league;
```

Comment:
I'm not familiar with reverse engineering packed binary data like this, but it seemed that:
4e = Premier League 
42 = Division 1
36 = Division 2
2a = Division 3
1e = Conference
00 = Non-english league

Reply:
This seems like a good guess, my 2002 savegame has 0x64 in that spot for my club, and it's in the premier league.

It is the only club with that number though, and all the other clubs seem to match your prediction here.

Reply:
After getting promoted from Div 2 to Div 1, my number is `0x49`, but all the other team's stayed the same, even the ones that got promoted/relegated. The important thing for the rest of the code is that it is not `0x00`, i.e. a player you can't buy.

Reply:
That's iteresting :) I'll see if I can gather some more samples on this.

Another way you can do this for your search for players is just looping the first 114 clubs in `gamea.club_index.all` (all the active clubs in the leagues, i'm guessing the last 4 aren't reachable via phone)